src/ensembl/src/shared/components/tabs/Tabs.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+import styles from './Tabs.scss';
+
+export type Tab = {
+  title: string;
+  isDisabled?: boolean;
+};
+
+export type TabsProps = {
+  tabs: Tab[];
+  classNames?: {
+    default?: string;
+    disabled?: string;
+    selected?: string;
+    tabsContainer?: string;
+  };
+  selectedTab: string;
+  selectTab: (selectedTab: string) => void;
+};
+
+export const Tabs = (props: TabsProps) => {
+  const [selectedTab, setSelectedTab] = useState(props.selectedTab);
```

Comment:
Why would this component keep state of selected tab? The parent knows which tab is selected (and passes it to this component as a prop), and knows what to do when a tab is changed — therefore, it looks like it's the responsibility of the parent to keep state of the selected tab.

Reply:
Removed the state from the tabs component.